README.md

```diff
@@ -70,3 +189,136 @@ or you can create a new user as admin
 ```sh
 User.create(email: email, password: password, admin: '1')
 ```
+
+## Run Rybyric
+
+In order to run Rubyric on your computer, you should start the server by typing 
+the following command.
+
+```sh
+bin/delayed_job start
+rails server 
+```
+
+Once the server is started, you can now initialise Rubyric at http://localhost:3000.
+
+If you are planning to run Rubyric along with A+. You may prefer to run Rubyric 
```

Comment:
Comma between the sentences: If you are planning to run Rubyric along with A+, you may prefer to run Rubyric...